yt_dlp/extractor/fujitv.py

```diff
@@ -16,19 +16,28 @@ class FujiTVFODPlus7IE(InfoExtractor):
     }
 
     _TESTS = [{
-        'url': 'https://fod.fujitv.co.jp/title/5d40/5d40810075',
+        'url': 'https://fod.fujitv.co.jp/title/5d40/5d40810076',
         'info_dict': {
-            'id': '5d40810075',
-            'title': '5d40810075',
+            'id': '5d40810076',
             'ext': 'mp4',
-            'format_id': '4000',
-            'thumbnail': 'http://i.fod.fujitv.co.jp/pc/image/wbtn/wbtn_5d40810075.jpg'
+            'title': '#1318 『まる子、まぼろしの洋館を見る』の巻',
+            'series': 'ちびまる子ちゃん',
+            'series_id': '5d40',
+            'description': 'md5:b3f51dbfdda162ac4f789e0ff4d65750',
+            'thumbnail': 'http://i.fod.fujitv.co.jp/img/program/5d40/episode/5d40810076_a.jpg',
         },
-        'skip': 'Expires after a week'
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
+        series_id = self._match_valid_url(url).group('sid')
+        self._download_webpage(url, video_id)
+        json_info = {}
+        if self._get_cookies(url).get('CT'):
+            token = self._get_cookies(url).get('CT').value
+            json_info = self._download_json('https://fod-sp.fujitv.co.jp/apps/api/episode/detail/?ep_id=%s&is_premium=false' % video_id, video_id, headers={'x-authorization': f'Bearer {token}'}, fatal=False)
```

Comment:
```suggestion
            json_info = self._download_json('https://fod-sp.fujitv.co.jp/apps/api/episode/detail/?ep_id=%s&is_premium=false' % video_id,
                video_id, headers={'x-authorization': f'Bearer {token}'}, fatal=False)
```
GH fucked up the formatting but suggestion is to break the line as it's too long